src/main/java/Duke.java

```diff
@@ -1,10 +1,131 @@
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        showStartMessage();
+        executeTillExit();
+        showExitMessage();
     }
+    private static void showStartMessage() {
+        String line = "_______________________________________________________________________________________________";
+        String logo =
+            "      ____                U _____ u   ____      ____        _" + System.lineSeparator() +
+            "     / __\"| u      ___    \\| ___\"|/U |  _\"\\ uU |  _\"\\ u U  /\"\\  u" + System.lineSeparator() +
+            "    <\\___ \\/      |_\"_|    |  _|\"   \\| |_) |/ \\| |_) |/  \\/ _ \\/" + System.lineSeparator() +
+            "     u___) |       | |     | |___    |  _ <    |  _ <    / ___ \\" + System.lineSeparator() +
+            "     |____/>>    U/| |\\u   |_____|   |_| \\_\\   |_| \\_\\  /_/   \\_\\" + System.lineSeparator() +
+            "      )(  (__).-,_|___|_,-.<<   >>   //   \\\\_  //   \\\\_  \\\\    >>" + System.lineSeparator() +
+            "     (__)      \\_)-' '-(_/(__) (__) (__)  (__)(__)  (__)(__)  (__)";
+
+        System.out.println("Hello from" + System.lineSeparator() + logo);
+        System.out.println(line);
+        System.out.println("Hello! I'm Sierra!" + System.lineSeparator() + "What can I do for you?");
+        System.out.println(line);
+    }
+
+    private static void executeTillExit() {
+        boolean isStop = false;
+        Scanner scannerObj = new Scanner(System.in);
+        ArrayList<Tasks> tasksAL = new ArrayList<>();
+
+        while (true) {
+            //User choice of what to do
+            System.out.println("I can do the following: Echo, List");
+            String userChoice = scannerObj.nextLine();
+
+            //Check if choice is valid
+            while (!checkValidInput(userChoice)) {
+                System.out.println("Invalid input, please try again. You may choose: Echo, List");
+                userChoice = scannerObj.nextLine();
+            }
+
+            //Execute choice
+            switch (userChoice.toLowerCase()) {
+            case "echo": //Echo what is said to user
+                System.out.println("I'm a copy cat!");
+                while (true) {
+                    String userInput = scannerObj.nextLine();
+                    if (userInput.equalsIgnoreCase("bye")) {
+                        isStop = true;
+                        break;
+                    } else if (userInput.equalsIgnoreCase("change")){
+                        break;
+                    } else {
+                        System.out.println(userInput);
+                    }
+                }
+                break;
+
+            case "list": //Activate list actions
```

Comment:
perhaps consider move "echo", " by" and etc. to constants? you might need these constants somewhere else..